test/fit_tests/common/fit_common.py

```diff
@@ -587,18 +587,23 @@ def get_node_sku(nodeid):
                 if VERBOSITY >= 2:
                     errmsg = "Error: SKU API failed {}, return code {} ".format(sku, skudata['status'])
                     print errmsg
+                    return "unknown"
         else:
-            if VERBOSITY >= 2:
-                errmsg = "Error: nodeid {} did not return a valid sku in get_rackhd_nodetype{}".format(nodeid,sku)
-                print errmsg
+            return "unknown"
```

Comment:
An unset sku is presented in the logs as an "unknown" sku string. This could be a null value, but it is easier and clearer as a text string, at least in my opinion.